src/common/types/varint.cpp

```diff
@@ -196,6 +196,10 @@ string Varint::VarIntToVarchar(const varint_t &blob) {
 	string decimal_string;
 	vector<uint8_t> byte_array;
 	bool is_negative;
+	if (blob.IsZero()) {
+		// zero is a bit of a special case because of 0 and -0
+		return "0";
```

Comment:
Instead of special casing it here maybe we can make `-0` illegal as a varint value?